README.rst

```diff
@@ -58,20 +58,69 @@ There are the available configuration options for the Password restriction:
 - **Password Explanation Text**: The text that will be displayed to the learner to explain how to access the content.
 - **Incorrect Password Explanation Text**: The text that will be displayed to the learner when the entered password is incorrect.
 
+IP Restriction
+--------------------
+When you select the IP restriction option, you need to specify the IP
+addresses that are allowed to access the content. Specify the IP addresses or
+IP ranges in the **IP Whitelist** field in the Content Restrictions component
+configuration.
+
+After publishing the unit with the Content Restrictions component, if the
+learner's IP address is not in the whitelist, the learner will see a message
+indicating that the content is restricted and will not be able to access the
+content. If the learner's IP address is in the whitelist, the content will be
+displayed.
+
+There are the available configuration options for the IP restriction:
```

Comment:
There's a typo here I didn't see in the docs. Could you help me fix it? Thanks!
```suggestion
Here are the available configuration options for the IP restriction:
```

Reply:
Done